osbs/conf.py

```diff
@@ -532,3 +532,26 @@ def get_equal_labels(self):
                 raise OsbsValidationException("Wrong equal_labels configuration")
 
         return equal_labels
+
+    def get_node_selector_platform(self):
+        nodeselector = {}
+
+        def assign_platform_to_key(key, val):
+            nodeselector[key.replace('node_selector.', '')] = val
+
+        for key in self.kwargs.keys():
+            if key.startswith('node_selector.'):
+                assign_platform_to_key(key, self.kwargs[key])
+        if self.args:
+            for key, val in self.args:
+                if key.startswith('node_selector.'):
+                    assign_platform_to_key(key, val)
+        try:
+            conf_items = self.scp.items(self.conf_section)
+            for key, val in conf_items:
+                if key.startswith('node_selector.'):
+                    assign_platform_to_key(key, val)
+        except configparser.Error:
+            return nodeselector
```

Comment:
I think we want kwargs/CLI args to override configuration values, so is this the wrong way around or should assign_platform_to_key() be using .setdefault() instead of assigning?